rosidl_typesupport_introspection_cpp/resource/msg__type_support.cpp.em

```diff
@@ -132,7 +143,7 @@ for index, member in enumerate(message.structure.members):
         # size_t string_upper_bound
         print('    0,  // upper bound of string')
         # const rosidl_generator_c::MessageTypeSupportHandle * members_
-        print('    NULL,  // members of sub message')
+        print('    nullptr,  // members of sub message')
```

Comment:
it is unrelated, but given that it's really just a two line change, I believe it can be changed within the scope of this PR as well.
I don't have any problem reverting this change here though if preferred.